Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -194,14 +194,12 @@ If not installing with Helm, here are the steps to install with manifests.
 Create the object store resources:
 
 1. Create the [external object store CR](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml) to configure connection to external gateways.
-2. Create an [Object store user](https://github.com/rook/rook/blob/master/deploy/examples/object-user.yaml) for credentials to access the S3 endpoint.
```

Comment:
So the object user that was created by this step was basically ignored since the other user was actually used?

Reply:
Updated